drop-in/src/main/java/com/adyen/checkout/dropin/DropInConfiguration.kt

```diff
@@ -291,6 +291,7 @@ class DropInConfiguration : Configuration, Parcelable {
          */
         fun addGooglePayConfiguration(googlePayConfiguration: GooglePayConfiguration): Builder {
             availablePaymentConfigs[PaymentMethodTypes.GOOGLE_PAY] = googlePayConfiguration
+            availablePaymentConfigs[PaymentMethodTypes.GOOGLE_PAY_LEGACY] = googlePayConfiguration
```

Comment:
I added this here, although it means that we are duplicating the entries, because the other way (only using `GOOGLE_PAY`) means that we'll need to make sure we always replace `GOOGLE_PAY_LEGACY` with `GOOGLE_PAY` when fetching the configuration (like in `availablePaymentConfigs` and `getConfigurationForPaymentMethod()`)

Reply:
Either way sounds good to me, I don't think it adds any memory usage.